--- ./java/src/IceInternal/OutgoingConnectionFactory.java.orig  2015-03-16 14:56:57.000000000 +1100
+++ ./java/src/IceInternal/OutgoingConnectionFactory.java       2015-03-16 14:58:10.000000000 +1100
@@ -29,7 +29,7 @@
         }
 
         public void
-        remove(K key, V value)
+        removeKeyValue(K key, V value)
         {
             java.util.List<V> list = this.get(key);
             assert(list != null);
@@ -622,9 +622,9 @@
             {
                 for(Ice.ConnectionI c : cons)
                 {
-                    _connections.remove(c.connector(), c);
-                    _connectionsByEndpoint.remove(c.endpoint(), c);
-                    _connectionsByEndpoint.remove(c.endpoint().compress(true), c);
+                    _connections.removeKeyValue(c.connector(), c);
+                    _connectionsByEndpoint.removeKeyValue(c.endpoint(), c);
+                    _connectionsByEndpoint.removeKeyValue(c.endpoint().compress(true), c);
                 }
             }
